scene/gui/menu_button.h

```diff
@@ -41,6 +41,7 @@ class MenuButton : public Button {
 	GDCLASS(MenuButton, Button);
 
 	bool clicked;
+	bool switch_on_hover;
```

Comment:
Don't know if this is the best name.
Suggestion `opens_automatically`

Reply:
I don't think this depicts the behaviour correctly.